BDArmory/Control/BDModulePilotAI.cs

```diff
@@ -1673,6 +1689,8 @@ bool RamTarget(FlightCtrlState s, Vessel v)
             return true;
         }
 
+        Vector3 staleTarget = Vector3.zero;
```

Comment:
Are these position and velocity of the stale target? I'd suggest renaming them `staleTargetPosition` and `staleTargetVelocity` so they're self-explanatory.